src/backend/storeManagers/storeManagerCommon/games.ts

```diff
@@ -70,49 +70,81 @@ const openBlankWindow = (options: BrowserWindowConstructorOptions) => {
   return browserGame.webContents
 }
 
+function handleUrlOpenForWebContents(
+  url: string,
+  contents: Electron.WebContents,
+  /* eslint-disable-next-line */
+  extensionImporter: any
```

Comment:
Why the any type here?

Reply:
It is an optional submodule :sweat_smile: `@hyperplay/extension-importer`